spec/ruby/core/io/timeout_spec.rb

```diff
@@ -0,0 +1,164 @@
+# -*- encoding: utf-8 -*-
+require_relative '../../spec_helper'
+
+ruby_version_is "3.2" do
+  describe "IO#timeout" do
+    before :each do
+      @fname = tmp("io_timeout.txt")
+      @file = File.open(@fname, "a+")
```

Comment:
**suggestion:** `@file` is used only in one test case so moving it out of the before/after callbacks would be a minor performance optimisation.

Reply:
Ah, true